src/helpers/FormatHelpers.ts

```diff
@@ -1,11 +1,53 @@
+import { 
+  camelCase,
+  pascalCase,
+  constantCase
+} from "change-case";
+
 export enum IndentationTypes {
   TABS = 'tabs',
   SPACES = 'spaces',
 }
 
 export class FormatHelpers {
   /**
-   * Ensures breaking text into new lines according to newline char (`\n`) in text
+   * Upper first char in given string value.
+   * @param {string} value to change
+   * @returns {string}
+   */
+  static upperFirst(value: string): string {
+    return value.charAt(0).toUpperCase() + value.slice(1);
+  }
+
+  /**
+   * Transform into a string with the separator denoted by the next word capitalized.
+   * @param {string} value to transform
+   * @returns {string}
+   */
+  static toCamelCase(value: string): string {
+    return camelCase(value);
+  }
+
+  /**
+   * Transform into a string of capitalized words without separators.
+   * @param {string} value to transform
+   * @returns {string}
+   */
+  static toPascalCase(value: string): string {
+    return pascalCase(value);
+  }
+
+  /**
+   * Transform into upper case string with an underscore between words.
+   * @param {string} value to transform
+   * @returns {string}
+   */
+  static toConstantCase(value: string): string {
+    return constantCase(value);
+  };
```

Comment:
Why these wrappers? 🤔

Reply:
I used them as plain functions, but I realized that in every renderer I use `FormatHelpers` class, so  putted them inside it.

Reply:
What I mean is, is there a reason for there to be a wrapper around the external functions?

Reply:
Nope, but I want to use them using `FormatHelpers` not with imports in each renderer :)

Reply:
Or I can change it to 

```ts
static toConstantCase = constantCase;
```

Reply:
That would be much better IMO 🎉